CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+# Contributing Guidelines
+
+### Welcome to airbnb/javascript!
+
+First off, thank you for considering contributing to Airbnb JavaScript Style Guide. It's people like you that make airbnb/javascript such a great tool.
+
+Following these guidelines helps to communicate that you respect the time of the developers managing and developing this open source project. In return, they should reciprocate that respect in addressing your issue, assessing changes, and helping you finalize your pull requests.
+
+### Bug reports
+
+A bug is a demonstrable problem that is caused by the code in the reposityory. Good bug reports are extremely helpful - thank you!
+
+Guidelines for bug reports: 
+
+1. **Use the GitHub issue search** &mdash; check if the issue has already been reported.
+
+2. **Check if the issue has been fixed** &mdash; try to reproduce it using the latest `master` or `next` branch in the repository.
+
+3. **Isolate the problem** &mdash; ideally create a reduced test case.
+
+A good bug report shouldn't leave others needing to chase you up for more information. Please try to be as detailed as possible in your report. What is your environment? What steps will reproduce the issue? What OS experiences the problem? What would you expect to be the outcome? All these details will help people to fix any potential bugs.
+
+Example: 
+
+
+> Short and descriptive example bug report title
+>
+> A summary of the issue and the browser/OS environment in which it occurs. If
+> suitable, include the steps required to reproduce the bug.
+>
+> 1. This is the first step
+> 2. This is the second step
+> 3. Further steps, etc.
+>
+> `<url>` - a link to the reduced test case
+>
+> Any other information you want to share that is relevant to the issue being
+> reported. This might include the lines of code that you have identified as
+> causing the bug, and potential solutions (and your opinions on their
+> merits).
+
+### Pull requests
+
+Good pull requests - patches, improvements, new features - are a fantastic
+help. They should remain focused in scope and avoid containing unrelated
+commits.
+
+**Please ask first** before embarking on any significant pull request (e.g.
+implementing features, refactoring code), otherwise you risk spending a lot of
+time working on something that the project's developers might not want to merge
+into the project.
```

Comment:
This is good advice for projects in general, but in this project, pretty much the only changes that are expected/welcome from non-maintainers are:
 - things with the [pull request wanted](https://github.com/airbnb/javascript/issues?q=is%3Aissue+is%3Aopen+label%3A%22pull+request+wanted%22) label
 - updating the eslint config to match the guide
 - editorial improvements to the guide

In particular, it should be noted that no "normative" (ie, things that alter the spirit of how you're recommended to write code) guide or eslint config changes will be made unless it's initiated by current Airbnb employees, so it's not a good use of time to propose any.